content/en/docs/usage/certificate.md

```diff
@@ -91,71 +90,159 @@ The `Certificate` will be issued using the issuer named `ca-issuer` in the
 
-This is configured using the `spec.privateKey.rotationPolicy` like so:
+Some issuers, like the built-in [Venafi
+issuer](/docs/configuration/venafi/), may disallow re-using private keys.
+If this is the case, you must explicitly configure the `rotationPolicy:
+Always` setting for each of your Certificate objects accordingly.
+
+{{% /alert %}}
+
+In the following example, the certificate has been set with
+`rotationPolicy: Always`:
 
 ```yaml
 apiVersion: cert-manager.io/v1
 kind: Certificate
-metadata:
-  name: my-cert
-  ...
 spec:
   secretName: my-cert-tls
-  dnsNames:
-  - example.com
   privateKey:
-    rotationPolicy: Always
+    rotationPolicy: Always # 🔰 Here.
+```
+
+### Actions that will trigger a rotation of the private key {#actions-triggering-private-key-rotation}
+
+Setting the `rotationPolicy: Always` or change the Certificate's spec won't
+rotate the private key immediately. In order to get the private key secret
+rotated, the certificate objects must be reissued. A certificate object is
+reissued with either:
+
+- when the X.509 certificate is nearing expiry,
+- when a reissuance is manually triggered with the following:
+  ```sh
+  kubectl cert-manager renew cert-1
+  ```
+  Note that the above command requires the [kubectl
+  cert-manager](/docs/usage/kubectl-plugin/#renew) plugin.
+
+{{% pageinfo color="warning" %}}
+
+**❌** Deleting the Secret resource associated with a Certificate resource is
+**not a recommended solution** for manually rotating the private key. The
+recommended way to manually rotate the private key is to trigger the reissuance
+of the Certificate resource with the following command (requires the [`kubectl
+cert-manager`](/docs/usage/kubectl-plugin/#renew) plugin):
+
+```sh
+kubectl cert-manager renew cert-1
 ```
 
-There are two supported rotation policies:
+{{% /pageinfo %}}
+
+### The `rotationPolicy` setting
+
+The possible values for `rotationPolicy` are:
+
+| Value                  | Description                                                   |
+| ---------------------- | ------------------------------------------------------------- |
+| `Never` (default)      | cert-manager reuses the existing private key on each issuance |
+| `Always` (recommended) | cert-manager regenerates a new private key on each issuance   |
+
+With `rotationPolicy: Never`, a private key is only generated if one does not
+already exist in the target Secret resource (using the `tls.key` key). All
+further issuances will re-use this private key. This is the default in order to
+maintain compatibility with previous releases.
+
+With `rotationPolicy: Always`, a new private key will be generated each time an
+action triggers the reissuance of the certificate object (see
+[actions](#actions-triggering-private-key-rotation) above). Note that if the
+private key secret already exists when creating the certificate object, the
+existing private key will not be used, since the rotation mechanism also
+includes the initial issuance.
```

Comment:
```suggestion
With `rotationPolicy: Always`, a new private key will be generated each time an
action triggers the reissuance of the certificate object (see
[Actions that will trigger a rotation of the private key](#actions-triggering-private-key-rotation) above). Note that if the
private key secret already exists when creating the certificate object, the
existing private key will not be used, since the rotation mechanism also
includes the initial issuance.
```